templates/parachain/README.md

```diff
@@ -47,90 +70,143 @@ git clone https://github.com/paritytech/polkadot-sdk-parachain-template.git para
 cd parachain-template
 ```
 
-### Build
+## Starting a Development Chain
+
+### Omni Node
 
-🔨 Use the following command to build the node without launching it:
+[Omni Node](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html) can
+be used to run the parachain template's runtime. `polkadot-omni-node` binary crate usage is described at a high-level
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+Please see the installation section at [`crates.io/omni-node`](https://crates.io/crates/polkadot-omni-node).
+
+#### Build `parachain-template-runtime`
 
 ```sh
-cargo build --release
+cargo build -p parachain-template-runtime --release
```

Comment:
```suggestion
cargo build --release
```

Right? it should be the same.

Reply:
just nit picking in the overall direction that `cargo build` should just build the runtime.